models/model.js

```diff
@@ -14,6 +15,10 @@ exports.selectReviews= () =>{
 
  exports.selectReviewByID=(id)=>{
     return db.query(`SELECT * FROM reviews WHERE review_id=${id};`).then((result)=>{
```

Comment:
you are potentially at risk of sql injection here, instead of passing the id straight in you want to use $1 and add the value to an array after the query ends, the notes should help with this. 

This will lead you to finding a different error code than you have checked for currently also, so check the err code in your error handlers and see what you get 😄